daphne_server/src/router/test_routes.rs

```diff
@@ -46,7 +46,7 @@ where
 
     router
         .route("/internal/delete_all", post(delete_all))
-        .route("/internal/test/ready", get(check_storage_readyness))
+        .route("/internal/test/ready", post(check_storage_readyness))
```

Comment:
Why must this be a POST?

Reply:
https://datatracker.ietf.org/doc/html/draft-dcook-ppm-dap-interop-test-design#section-4.3.1